### ISSUE TITLE
Unaligned pointer access

### ISSUE THREAD

**Issue by mgerhardy** [assimp/assimp#1057]
On x86 this is usually only slow - but it might even crash on other platforms.

```
In file included from /home/mgerhardy/dev/engine/contrib/libs/assimp/include/assimp/Importer.hpp:54:
/home/mgerhardy/dev/engine/contrib/libs/assimp/include/assimp/types.h:208:19: warning: taking address of packed member 'r' of class or structure 'aiColor3D' may result in an unaligned pointer value [-Waddress-of-packed-member]
        return *(&r + i);
                  ^
/home/mgerhardy/dev/engine/contrib/libs/assimp/include/assimp/types.h:213:19: warning: taking address of packed member 'r' of class or structure 'aiColor3D' may result in an unaligned pointer value [-Waddress-of-packed-member]
        return *(&r + i);
                 
```

found by clang version 4.0.0-svn279916-1 (trunk)

**Comment by kimkulling** [comment]
Closed by d4223d1fce2e9edd8d4c1be1fa48f70042b71876

**Comment by msc-** [comment]
This issue just started cropping up with the newly released Xcode 8.3. I guess they must have update LLVM.

I used MacPorts to install assimp. It's latest version, which I have, is 3.0.1270_1. How to I get the MacPorts version updated with the fix in d4223d1?
